mantidproject/mantid
# Add additional sample logs in indirect routines

Fixes #11649.

To test:
- Run ConvFit (`irs26176_graphite002_red`, `irs26173_graphite002_res`, One Lorentzian), see that `sam_workspace` and `res_workspace` logs exist
- Code review

This is only a minor issue so is not worth a mention in the release notes.
